dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/DefaultArguments.java

```diff
@@ -124,6 +124,47 @@ public String valuePattern() {
     }
   }
 
+  public enum HadoopSaslQop {
+    AUTHENTICATION("auth"),
+    INTEGRITY("auth-int"),
+    PRIVACY("auth-conf");
+
+    public final String qopValue;
+
+    HadoopSaslQop(String qopValue) {
+      this.qopValue = qopValue;
+    }
+  }
+
+  public static class HadoopSaslQopValueConverter implements ValueConverter<String> {
+
+    public static HadoopSaslQopValueConverter INSTANCE = new HadoopSaslQopValueConverter();
+
+    private HadoopSaslQopValueConverter() {}
+
+    @Override
+    public String convert(String value) {
+      for (HadoopSaslQop qop : HadoopSaslQop.values()) {
+        if (qop.qopValue.equals(value) || qop.name().equalsIgnoreCase(value)) {
+          return qop.qopValue;
+        }
+      }
+      throw new ValueConversionException("Not a valid QOP: " + value);
```

Comment:
I would suggest throwing `MetadataDumperUsageException` to make this converter independent of joptsimple library.